python/pytorchserver/pytorchserver/model.py

```diff
@@ -68,6 +67,6 @@ def predict(self, request: Dict) -> Dict:
             raise Exception(
                 "Failed to initialize Torch Tensor from inputs: %s, %s" % (e, inputs))
         try:
-            return { "predictions":  self._pytorch(inputs).tolist() }
+            return {"predictions":  self._pytorch(inputs).tolist()}
```

Comment:
Curious what editor you're using that's complaining about this where mine isnt.

Reply:
pycharm :)